src/v0/destinations/user/transform.js

```diff
@@ -76,9 +82,8 @@ const trackResponseBuilder = async (message, destination) => {
     return responseBuilder(payload, endpoint, method, apiKey);
   }
 
-  throw new CustomError(
-    "[ User.com ]:: No user found with given lookup field, Track event cannot be completed if there is no valid user",
-    400
+  throw new NetworkInstrumentationError(
+    "[User.com] :: No user found with given lookup field, Track event cannot be completed if there is no valid user"
```

Comment:
try not put [User.com] :: these in error messages

Reply:
updated